Spanner/tests/System/AdminTest.php

```diff
@@ -127,10 +129,74 @@ public function testDatabase()
         $this->assertEquals($db->ddl()[0], $stmt);
     }
 
-    public function testPgDatabase()
+    public function testCustomerManagedInstanceConfigurations()
```

Comment:
Can we split these tests into individual functions?

Reply:
Do you mean individual tests, or just split them into a number of different private functions to separate them more clearly? Both would be possible, but I'm a little reluctant to do the former. That would mean that the integration tests would take even longer, as we would have to create a new custom instance config for each test case, which takes some time.

Reply:
I do mean the former.
But just to clarify you won't need to create an instance config for every test.

You could declare a static variable and once the instance config is created, you assign it to the variable.
Your other tests should `@depend` on the creation test.

This is the pattern we have followed in a lot of our test files.